### PR TITLE
[GSOC'24 Amharic chapter] Extend Existing Extractors For Amharic 

### PR DESCRIPTION
**Pull request by Meti-Adane**
- Added Amharic configurations to extend existing extractors 
- Added associated tests
- Included extended extractors in property files

### CODE REVIEW

File: core/src/main/scala/org/dbpedia/extraction/config/dataparser/ParserUtilsConfig.scala [hunk 1]
```diff
@@ -14,6 +14,23 @@ object ParserUtilsConfig
...
+            "billion" -> 9,
+            "ቢሊዮን" -> 9,
+            "bln" -> 9,
+            "trillion" -> 12,
+            "ትሪሊዮን" -> 12,
+            "quadrillion" -> 15,
+            "ኳድሪሊየን" -> 15
         ),
         // For "ar" configuration, rendering right-to-left may seems like a bug, but it's not.
         // Don't change this else if you know how it is done.
```
Comment (TallTed) [inline core/src/main/scala/org/dbpedia/extraction/config/dataparser/ParserUtilsConfig.scala:36]:
```suggestion
        // Don't change this unless you know how it works.
```

File: core/src/main/scala/org/dbpedia/extraction/config/mappings/DateIntervalMappingConfig.scala [hunk 2]
```diff
@@ -11,6 +11,7 @@ object DateIntervalMappingConfig
     // Don't change this else if you know how it is done.
```
Comment (TallTed) [inline core/src/main/scala/org/dbpedia/extraction/config/mappings/DateIntervalMappingConfig.scala:11]:
```suggestion
    // Don't change this unless you know how it works.
```

**Review by TallTed — CHANGES_REQUESTED** [review]
Human facing. Small stuff.

**Review by jimkont — APPROVED** [review]
looks great, we should also incorporate @TallTed 's documentation suggestions

**Pull request merged**